source/ocsp-support/ocsp-support.rst

```diff
@@ -100,9 +100,12 @@ invalid, the driver SHOULD end the connection.
     attempt to validate the status of the unvalidated certificates
     using the cached CRLs.
 
-7.  If the server’s certificate remains unvalidated and that certificate
-    has an OCSP endpoint, the driver SHOULD reach out to the OCSP
-    endpoint specified and attempt to validate that certificate.
+7.  If the server's certificate remains unvalidated and that certificate
+    has a list of OCSP responder endpoints, the driver SHOULD send HTTP requests
+    to the responders in parallel. The first valid response that concretely marks
+    the certificate status as good or revoked should be used. A five-second timeout
+    SHOULD be used for the requests. The status for a response should only be checked
+    if there are no errors parsing it and it is correctly signed.
```

Comment:
I think what you're saying here is any response that fails https://tools.ietf.org/html/rfc6960#section-3.2 will be ignored? I've been treating failures in section 3.2 the same as REVOKED. This seems to be implying those failures should be treated as UNKNOWN? If so, that feels like a fundamental change to this spec, not just this verification step.

Reply:
Resolving as we discussed this offline. The new commits explicitly state the verification constraints that we discussed during our meeting.